config/environments/development.rb

```diff
@@ -4,7 +4,7 @@
   # In the development environment your application's code is reloaded on
   # every request. This slows down response time but is perfect for development
   # since you don't have to restart the web server when you make code changes.
-  config.cache_classes = false
+  config.cache_classes = true
```

Comment:
This configuration must not be the default value as it will complexify software development. You may have to use a environment variable to configure this only if running with vagrant.